### PR TITLE
Html templating finished

### PR DESCRIPTION
**Pull request by bi6o**

### CODE REVIEW

File: dev/view/groups.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="tab-pane fade" id="tab-groups">
+    <!--include 'groups/navigation.html' -->
+    <div class="tab-content inner-tab-content">
+        <!--include 'groups/public.html' -->
+        <!--include 'groups/private.html' -->
+        <!--include 'groups/create.html' -->
+        <!--include 'groups/manage.html' -->
+    </div>
+</div>
```
Comment (abhisaha1) [inline dev/view/groups.html:9]: This whole file should be inside the template, `popup_dev.html`. Nested templates will make the build process more complicated. Please move the html of this file inside `popup_dev.html`.

File: dev/view/groups/navigation.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ul class="nav nav-pills nav nav-pills secondary-nav">
+    <li class="active"><a href="#tab-public-groups" class="pill-item" data-toggle="pill">Public</a></li>
+    <li><a href="#tab-private-groups" class="pill-item" data-toggle="pill">Private</a></li>
+    <li><a href="#tab-create-groups" class="pill-item" data-toggle="pill">Create</a></li>
+    <li><a href="#tab-manage-groups" class="pill-item" data-toggle="pill">Manage</a></li>
+</ul>
```
Comment (abhisaha1) [inline dev/view/groups/navigation.html:6]: This file can also be inside `popup_dev.html`.  We basically want our `popup_dev.html` to be like a template containing the structure of our application. We just want to separate the views.

File: dev/view/header.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div class="header">
+    <div class="pull-left">
+        Group: &nbsp;&nbsp;
+    </div>
+    <div class="pull-left">
+        <select id="groups-dd"></select>
+    </div>
+    <span class="clearfix" />
+</div>
```
Comment (abhisaha1) [inline dev/view/header.html:9]: Same goes here. Should be a part of `feed.html`.

File: dev/view/links.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<div class="tab-pane fade" id="tab-links">
+    <!--include 'links/navigation.html' -->
+    <div class="tab-content inner-tab-content">
+        <!--include 'links/favourites.html' -->
+        <!--include 'links/sent.html' -->
+    </div>
+</div>
```
Comment (abhisaha1) [inline dev/view/links.html:7]: Should be a part of `popup_dev.html`

File: dev/view/links/favourites.html [hunk 5]
```diff
@@ -0,0 +1,5 @@
+<div class="tab-pane active" id="tab-favourites">
+    <div class="scroll">
+        <div class="items  user-enabled"></div>
+    </div>
+</div>
```
Comment (abhisaha1) [inline dev/view/links/favourites.html:5]: Should be a part of `popup_dev.html`

File: dev/view/links/navigation.html [hunk 6]
```diff
@@ -0,0 +1,5 @@
+<ul class="nav nav-pills nav nav-pills secondary-nav">
+    <li class="active"><a href="#tab-favourites" class="active pill-item" data-toggle="pill">Favourites</a></li>
+    <li><a href="#tab-sent" class="pill-item" data-toggle="pill">Sent</a></li>
+    <!-- <li><a href="#tab-received" class="pill-item" data-toggle="pill">Received</a></li> -->
+</ul>
```
Comment (abhisaha1) [inline dev/view/links/navigation.html:5]: Should be a part of `popup_dev.html`

File: dev/view/links/sent.html [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<div class="tab-pane" id="tab-sent">
+    <div class="scroll">
+        <div class="items"></div>
+    </div>
+</div>
```
Comment (abhisaha1) [inline dev/view/links/sent.html:5]: Should be a part of `popup_dev.html`

File: dev/view/modals/email.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+            </div>
+        </div>
+        <!-- /.modal-content -->
+    </div>
+    <!-- /.modal-dialog -->
+    <<<<<<< HEAD </div>
+        =======
+</div>
+>>>>>>> e781d1145f1259e4388618147ba32c7cbc6d2dd1
```
Comment (abhisaha1) [inline dev/view/modals/email.html:41]: Some problem with line 38 and 41. Please remove them

File: dev/view/navigation.html [hunk 9]
```diff
@@ -0,0 +1,8 @@
+<ul class="nav nav-tabs main-nav">
+    <li class="active"><a href="#tab-notifications" data-toggle="tab">Notifications</a><span class="short-border" /></li>
+    <li class="authorized"><a href="#tab-feed" data-toggle="tab">Feed</a><span class="short-border" /></li>
+    <li class="authorized"><a href="#tab-post" data-toggle="tab">Post</a><span class="short-border" /></li>
+    <li class="authorized"><a href="#tab-links" data-toggle="tab">Links</a><span class="short-border" /></li>
+    <li class="authorized"><a href="#tab-groups" data-toggle="tab">Groups</a><span class="short-border" /></li>
+    <li><a href="#tab-settings" data-toggle="tab">Settings</a><span class="short-border" /></li>
+</ul>
```
Comment (abhisaha1) [inline dev/view/navigation.html:8]: Should be a part of `popup_dev.html`

File: dev/view/notification.html [hunk 10]
```diff
@@ -0,0 +1,5 @@
+<div class="scroll tab-pane fade active in" id="tab-notifications">
+    <div class="items user-enabled">
+        <div id="loader" class="preloader"><img src="public/images/loader.svg"></div>
+    </div>
+</div>
```
Comment (abhisaha1) [inline dev/view/notification.html:5]: Should be a part of `popup_dev.html`

File: dev/view/settings.html [hunk 11]
```diff
@@ -0,0 +1,8 @@
+<div class="tab-pane fade" id="tab-settings">
+    <!--include 'settings/navigation.html' -->
+    <div class="tab-content inner-tab-content">
+        <!--include 'settings/profile.html' -->
+        <!--include 'settings/customize.html' -->
+        <!--include 'settings/about.html' -->
+    </div>
+</div>
```
Comment (abhisaha1) [inline dev/view/settings.html:8]: Should be a part of `popup_dev.html`

File: dev/view/settings/navigation.html [hunk 12]
```diff
@@ -0,0 +1,5 @@
+<ul class="nav nav-pills nav nav-pills secondary-nav">
+    <li class="active"><a href="#tab-profile" class="pill-item" data-toggle="pill">Profile</a></li>
+    <li><a href="#tab-customize" class="pill-item" data-toggle="pill">Customize</a></li>
+    <li><a href="#tab-about" class="pill-item" data-toggle="pill">About</a></li>
+</ul>
```
Comment (abhisaha1) [inline dev/view/settings/navigation.html:5]: Should be a part of `popup_dev.html`

File: dev/view/system-messages.html [hunk 13]
```diff
@@ -0,0 +1 @@
+<div id="msg" class="hide"></div>
```
Comment (abhisaha1) [inline dev/view/system-messages.html:1]: Should be a part of `popup_dev.html`

**Review by abhisaha1 — CHANGES_REQUESTED** [review]
Thanks @bi6o , this is awesome. There are some pages that you created, which should be written directly in `popup_dev.html`. This is important, because we do not want nested templates. Since we don't have a framework, the build process will become very complex. Our `popup_dev.html` should be the only template containing `<!-- include xxxx -->` and no other file should contain includes.

When I thought of the build, this is the problem I faced.
- Parse `popup_dev.html` and read all includes inside 
- Get content of all includes . 
- Then further parse all includes and read all includes inside them.
- This can go till any depth.

Just to limit this, lets try to keep it 1 level deep.

File: dev/view/settings.html (hunk 11 above)
Comment (bi6o) [reply dev/view/settings.html]: Do you mean a part of the settings.html file? as in the sub-templates should be in that file and not separated (still one depth)?

File: dev/view/header.html (hunk 3 above)
Comment (bi6o) [reply dev/view/header.html:9]: For this one, I thought the header could be reusable if another tab in the future needed that header in it. Will move it back to feed

**Pull request closed without merging**